### DISCUSSION TITLE
qdrant memory optimization

### DISCUSSION THREAD

**Discussion by zhangzf875** [qdrant/qdrant#4584] [General]
I deployed qdrant1.7.2 on linux, and after using it for some time, I found that the memory increased with the collection and points, and the memory increased a little bit; Later, I changed the configuration of the index to store on disk, and found that the memory was reduced from 22G to 11.7G. Now I changed the memory configuration from storage on disk to storage in memory, and found that the memory did not change from 11.7G to about 22G. What is the cause
How to modify:
curl -X PATCH http://localhost:6333/collections/${collections} \
-H 'Content-Type: application/json' \
--header 'api-key: d37c985d343b876f6b1bce8a03ea3ae5'  \
-d '{
"hnsw_config": {
"on_disk": false
}
} '

**Comment by generall** [comment]
Memory consumption is not a trivial topic, it doesn't make sense to discuss without detailed definition of what we consider "consumption". 
Please refer to this article - https://qdrant.tech/articles/memory-consumption/ for more info
